### PR TITLE
feat: proper home assistant add configuration

### PR DESCRIPTION
**Pull request by t0bst4r**

### CODE REVIEW

**Comment by t0bst4r** [comment]
@genehand i created the PR for you to be able to write comments 😄

File: matterbridge/run.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+HOME_ASSISTANT_URL=http://supervisor/core
+HOME_ASSISTANT_ACCESS_TOKEN=$SUPERVISOR_TOKEN
+HOME_ASSISTANT_CLIENT_CONFIG=$(bashio::config 'home_assistant_client_config')
+
+export HOME_ASSISTANT_URL
+export HOME_ASSISTANT_ACCESS_TOKEN
+export HOME_ASSISTANT_CLIENT_CONFIG
+
+if [ ! -d /root/.matterbridge/storage ]; then
```
[the remark this answers is not in the document]
Comment (genehand) [reply matterbridge/run.sh:11]: 1. Storing the config in the container means that starting a fresh container will reset everything.  You'd have to re-pair the matter controller for upgrades, etc.  I added a symlink in the dockerfile from `/root/.matterbridge` to `/config` which is in turn mounted from `addon_config`. 
2. Cool that sounds cleaner 😄

File: repository.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+name: Matterbridge
+url: https://github.com/genehand/matterbridge-home-assistant-addon
+maintainer: Tobias Glatthar
```
[the remark this answers is not in the document]
Comment (genehand) [reply repository.yaml:3]: 🙈

**Comment by genehand** [comment]
I didn't see this one before adding https://github.com/t0bst4r/matterbridge-home-assistant-addon/pull/2 - I'll update over there since it includes a couple other tweaks

**Comment by t0bst4r** [comment]
I’ll close this one

**Pull request closed without merging**